haskell/src/Monocle/Backend/Queries.hs

```diff
@@ -146,11 +146,11 @@ getEventCounts query =
     selfMergedQ = mkAnd [query, BH.TermQuery (BH.Term "self_merged" "true") Nothing]
 
 -- $setup
--- >>> import Data.Aeson (encode)
+-- >>> import Data.Aeson (encode, decode)
 
 -- | The histo event query
--- >>> putTextLn $ decodeUtf8 $ encode histoEventAgg
--- {"date_histogram":{"interval":"day","field":"created_at"}}
+-- >>> decode $ encode histoEventAgg :: Maybe (Map Text Value)
+-- Just (fromList [("date_histogram",Object (fromList [("field",String "created_at"),("interval",String "day")]))])
```

Comment:
It seems like this key order issue remain, and using aeson-pretty may prevent that. Here is an example: https://github.com/softwarefactory-project/matrix-client-haskell/commit/4ca4963cfd06379d9bdce49742af854aed6a0d37 . Could you try something similar here, and we'll compare our result?

Reply:
Ok I'll see for aeson-pretty